justfiles/deploy.just

```diff
@@ -23,17 +23,17 @@ _deploy-eigen-handler-inner HANDLER_ADDRESS:
         --rpc-url "{{ETHEREUM_RPC_URL}}" \
         --broadcast \
         --json 2>&1 > /dev/null
-    @cat broadcast/Deploy.s.sol/31337/run-latest.json | jq -r '.transactions[0].contractAddress' | tr -d '\n' > "{{CONTRACT_ADDRESS_FILEPATH}}"
+    cat broadcast/Deploy.s.sol/31337/run-latest.json | jq -r '.transactions[0].contractAddress' | tr -d '\n' > "{{CONTRACT_ADDRESS_FILEPATH}}"
 
 _deploy-eigen-manager:
-    @wavs-cli deploy-eigen-service-manager --home ./config --chain anvil --eth-mnemonic "{{ETHEREUM_MNEMONIC}}" \
+    @{{WAVS_CMD}} wavs-cli deploy-eigen-service-manager --home=/data/config --chain anvil --eth-mnemonic "{{ETHEREUM_MNEMONIC}}" \
 
 upload-components:
-    @just _upload-component "{{justfile_directory()}}/{{WASI_OUT_DIR}}/prepare.wasm" {{DIGEST_PREPARE_FILEPATH}}
-    @just _upload-component "{{justfile_directory()}}/{{WASI_OUT_DIR}}/commit.wasm" {{DIGEST_COMMIT_FILEPATH}}
+    @just _upload-component "/data/.build-artifacts/wasi-components/prepare.wasm" .build-artifacts/component.prepare.digest
+    @just _upload-component "/data/.build-artifacts/wasi-components/commit.wasm" .build-artifacts/component.commit.digest
```

Comment:
Some commands were not happy with variables (like here and also with the service deploy @{{FILENAME}} while others were. not sure why or what the context is, which is why this is a little messy

if you want to play around with it more than happy for you too

Reply:
nah, really appreciate you diving and and making this all work, whatever makes the most sense for you is good with me